dcatregister/netcdf.py

```diff
@@ -27,12 +28,20 @@ def get_spatial_info(data):
     return spatial_dict
 
 def get_temporal_info(data):
-    return
+    t = dict()
+    date_s = str(data.datetime[0]).split('\'')
+    date_e = str(data.datetime[-1]).split('\'')
```

Comment:
It is not clean code because splitting can generate a exception.

Please refer to array docs
The type of data.datetime[0] is http://xarray.pydata.org/en/stable/generated/xarray.DataArray.html

```bash
    date_s = str(data.datetime[0].data)
    date_e = str(data.datetime[-1].data)
```

Please handle the following corner cases:
- when the nc doesn't contain the attribute `data.datetime` 
- the length of the data.datatime is zero

Reply:
addressed